File: c/3-operatory.c
#include <stdio.h>

int main()
{
	int liczba1=2, liczba2=3;

	printf("%d + %d=%d\n", liczba1, liczba2, liczba1 + liczba2);
	printf("%d - %d=%d\n", liczba1, liczba2, liczba1 - liczba2);
	printf("%d * %d=%d\n", liczba1, liczba2, liczba1 * liczba2);
	printf("%d / %d=%d\n", liczba1, liczba2, liczba1 / liczba2);
	printf("%d / %d=%f\n", liczba1, liczba2, 1.0 * liczba1 / liczba2);

	return 0;
}

File: objc/3-operatory.m
#import <Foundation/Foundation.h>

int main()
{
	NSAutoreleasePool *pool = [[NSAutoreleasePool alloc] init];
	int liczba1=2, liczba2=3;

	NSLog(@"%d + %d=%d\n", liczba1, liczba2, liczba1 + liczba2);
	NSLog(@"%d - %d=%d\n", liczba1, liczba2, liczba1 - liczba2);
	NSLog(@"%d * %d=%d\n", liczba1, liczba2, liczba1 * liczba2);
	NSLog(@"%d / %d=%d\n", liczba1, liczba2, liczba1 / liczba2);
	NSLog(@"%d / %d=%f\n", liczba1, liczba2, 1.0 * liczba1 / liczba2);
	[pool drain];

	return 0;
}
